manifests/stage-macvlan-cni/0010-macvlan-ds.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: macvlan-cni-ds
+  namespace: {{ .RuntimeSpec.Namespace }}
+  labels:
+    tier: node
+    app: macvlan-cni
+spec:
+  selector:
+    matchLabels:
+      name: macvlan-cni
+  template:
+    metadata:
+      labels:
+        name: macvlan-cni
+        tier: node
+        app: macvlan-cni
+    spec:
+      hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: macvlan-cni
+          image: {{ .CrSpec.Repository }}/{{ .CrSpec.Image }}:{{ .CrSpec.Version }}
```

Comment:
do we have a default image?

Reply:
no, Didn't find any, Do you prefer to push one to mellanox repo?

Reply:
yes, we need to build one push it to mellanox docker. v0.8.7 version

Reply:
I work on it

Reply:
So, based on discussions in PR#47,  this will change to `container-networking-plugins` and deploy all plugins that come with this package.

Reply:
changed